README.md

```diff
@@ -85,19 +85,73 @@ The `href` can contain an optional server placeholder `{s}`. If `{s}` is used, t
 
 Links to a [PMTiles](https://github.com/protomaps/PMTiles/blob/main/spec/v3/spec.md) file (versions 3.x).
 
-| Field Name      | Type                 | Description |
-| --------------- | -------------------- | ----------- |
-| rel             | string               | **REQUIRED**. Must be set to `pmtiles`. |
-| href            | string               | **REQUIRED**. Link to a PMTiles file (usually ends with `.pmtiles`). |
-| type            | string               | Recommended to be set to `application/vnd.pmtiles`. |
-| pmtiles:layers  | \[string]            | For vector tiles, the layers to show on the map by default. If not provided, it's up to the discretion of the implementation to choose a layer from the `vector_layers` in the PMTiles metadata. |
+| Field Name               | Type                                           | Description |
+| ------------------------ | ---------------------------------------------- | ----------- |
+| rel                      | string                                         | **REQUIRED**. Must be set to `pmtiles`. |
+| href                     | string                                         | **REQUIRED**. Link to a PMTiles file (usually ends with `.pmtiles`). |
+| type                     | string                                         | Recommended to be set to `application/vnd.pmtiles`. |
+| pmtiles:layers           | \[string]                                      | For vector tiles, the layers to show on the map by default. If not provided, it's up to the discretion of the implementation to choose a layer from the `vector_layers` in the PMTiles metadata. |
+| pmtiles:layer_properties | Map\<string, Map\<string, JSON Schema Object>> | For vector tiles, the properties/fields available for each layer and their corresponding [JSON Schema](#json-schema-object) as an object. |
 
 The [Tile Type](https://github.com/protomaps/PMTiles/blob/main/spec/v3/spec.md#tile-type-tt) of the
 PMTiles data source can be read from the first 127 bytes of the the binary header.
 
 It is typical to assume a tile size of 256x256 pixels for raster tiles and 512x512 pixels for vector tiles,
```

Comment:
clarify this is "display (CSS) pixels" as appears on screen, which is different than device pixels (retina, etc)

Reply:
Moved into a separate issue as it has no relation to this PR: https://github.com/stac-extensions/web-map-links/issues/21